src/server.js

```diff
@@ -1,8 +1,17 @@
 'use strict';
 var http = require('http');
+var publish = require('./publish');
+var subscribe = require('./subscribe');
+
 var port = process.env.PORT || 1337;
+console.log('process.env.PORT =' + process.env.PORT);
+console.log('process.env.RABGLITMQ =' + process.env.RABGLITMQ);
```

Comment:
@snorret : This environment variable RABGLITMQ should be set to our real RMQ cluster url for the docker environment